WordPress/Classes/ViewRelated/Me/All Domains/Views/AllDomainsListActivityIndicatorTableViewCell.swift

```diff
@@ -24,9 +24,13 @@ final class AllDomainsListActivityIndicatorTableViewCell: UITableViewCell {
         fatalError("init(coder:) has not been implemented")
     }
 
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        self.activityIndicator.startAnimating()
+    }
```

Comment:
This fixes an error where the activity indicator wasn't animating when tapping the "Try again" button of the empty state view.